pipeline/src/types/index.ts

```diff
@@ -35,6 +35,17 @@ export type { PrismicImage };
 import { Article, Contributor, ArticleFormat, Image } from "./transformed";
 export type { Article, Contributor, ArticleFormat, Image };
 
+export type ElasticsearchArticle = {
+  id: string;
+  display: Article;
+  query: {
+    title: string;
+    published: Date;
+    contributors: string[];
+    promo_caption?: string;
```

Comment:
I would expect this to have the same name that it does in the display model - is there a reason it's different?

Reply:
Happy to change that, actually wanted to and had already approved it with Harrison (these are the names in his prototype), so changing it now.